src/FieldWarning/Assets/Util/ConfigReader.cs

```diff
@@ -20,7 +20,7 @@ namespace PFW
 {
 	public static class ConfigReader
     {
-        public static Unit FindUnit(string unitId)
+        public static Unit ParseUnit(string unitId)
         {
             MatchCollection matches =
                     Regex.Matches(unitId, @"^(Unit__)([A-Z]+)(?:--)([-_a-zA-Z0-9]+)");
```

Comment:
we can totally get rid of this type of regex matching by making a more flexible json system as mentioned above.  Would anything ever be not a unit?

Reply:
done